mod/state-transition/pkg/core/state_processor_staking_test.go

```
@@ -144,30 +152,279 @@
+	)
+
+	newEpochVals, err := sp.Transition(ctx, beaconState, blk)
+	require.NoError(t, err)
+	require.Len(t, newEpochVals, len(genDeposits)) // just topped up one validator
+
+	// Assuming genesis order is preserved here which is not necessary
+	// TODO: remove this assumption
+
+	// all genesis validators other than the last are unchanged
+	for i := range len(genDeposits) - 1 {
+		require.Equal(t, genVals[i], newEpochVals[i], fmt.Sprintf("idx: %d", i))
```

golangci-lint (1.62.2)

formatter: remove unnecessary fmt.Sprintf

(testifylint)